tuning/sft_trainer.py

```diff
@@ -515,7 +515,7 @@ def main(**kwargs):  # pylint: disable=unused-argument
             exp_metadata,
         )
     except Exception as e:  # pylint: disable=broad-except
-        logging.error(traceback.format_exc())
+        logging.get_logger("error").error(traceback.format_exc())
```

Comment:
Just curious what's this change for ?

Reply:
`logging.error` isn't part of `transformers.utils.logging` .


For example, when you run this:

```bash
 python -c 'from transformers.utils import logging; logging.error("foo")'
```

python raises the following exception:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'transformers.utils.logging' has no attribute 'error'
```

so I just created a logger object called "error" and had that print the traceback. I assume that this line of code was originally using the `logging` module for which `logging.error("foo")` works

Reply:
Unfortunately, I don't remember how I made sft_trainer.py execute that code.